ups/sims_selfcal.table

```diff
@@ -11,9 +11,7 @@ setupRequired(swig)
 envSet(MODTRAN_EXECUTABLE, ${PRODUCT_DIR}/modtran/Mod90_5.2.1.exe)
 envSet(MODTRAN_DATADIR, ${PRODUCT_DIR}/modtran/DATA/)
 
-envAppend(LD_LIBRARY_PATH, ${PRODUCT_DIR}/lib)
-envAppend(DYLD_LIBRARY_PATH, ${PRODUCT_DIR}/lib)
```

Comment:
The above two lines are the only cases where envAppend is NOT replaced with envPrepend but removed entirely, since this is a pure-Python package.